### PR TITLE
Migrated CPAP ordering app to new URL

### PR DESCRIPTION
**Pull request by carlosfelix2**
## Summary
Continue the migration of the CPAP React app from /health-care/order-hearing-aid-or-CPAP-supplies-form to /my-health/order-hearing-aid-or-cpap-supplies-form.
This change makes it so the CPAP ordering app is now at the new URL. This in conjunction with the two entries in the `content-build` registry is that the app can be accessed by using either URL, but the URL in the browser will change to the new URL specified in the manifest.json of the app. This essentially acts as a redirect from one old URL to the new one. The next step after this is to make the redirect permanent via the reverse proxy.
This PR also deletes the redirect app that is no longer used.

## Related issue(s)
https://app.zenhub.com/workspaces/mhv-on-vagov-top-level-view-62619a987d74510018ecc546/issues/gh/department-of-veterans-affairs/va.gov-team/70757

## Testing done
Manual testing
- Verify the CPAP ordering app can be accessed at both `/health-care/order-hearing-aid-or-CPAP-supplies-form` and `/my-health/order-hearing-aid-or-cpap-supplies-form` URLs
- Verify the browser redirects to the new URL when accessing the CPAP ordering app with the old URL.

## Screenshots
N/A

## What areas of the site does it impact?
- CPAP ordering app

## Acceptance criteria
Migrate the CPAP app from `/health-care/order-hearing-aid-or-CPAP-supplies-form` to `/my-health/order-hearing-aid-or-cpap-supplies-form`

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

### CODE REVIEW

File: src/applications/registry.scaffold.json [hunk 1]
```diff
@@ -132,11 +132,6 @@
     "widgetType": "pension-how-do-i-apply-widget",
     "rootUrl": "/pension-how-do-i-apply-widget"
   },
-  {
```
Comment (carlosfelix2) [inline src/applications/registry.scaffold.json:135]: Note: This file is no longer used by the system or build. Unknown why this has not been deleted.

**Review by radavis — APPROVED** [review]
LGTM!f

**Review by jzucadi — APPROVED** [review]
Good to go

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by dcloud — APPROVED** [review]
👍

**Comment by carlosfelix2** [comment]
Waiting for other changes

**Pull request closed without merging**